.travis.yml

```diff
@@ -1,27 +1,77 @@
+sudo: false
+
 language: php
 
-php:
-  - 5.5
-  - 5.6
+cache:
+  directories:
+    - $HOME/.composer/cache
+    - vendor
 
 env:
-  - MONGO_DRIVER=mongo
+  global:
+    - COMPOSER_ARGS="--no-interaction --ignore-platform-reqs"
```

Comment:
I know it's a bit more complicated with the composer bug making the installation of mongo-php-adapter more complicated, but I'd prefer we don't have the `ignore-platform-reqs` option set.